components/TabItem/TabItem.module.css

```diff
@@ -71,6 +75,10 @@
 }
 
 @media (min-width: 768px) {
+  .hiddenMobile {
+    display: inline;
```

Comment:
Maybe `initial` is a better value here. Otherwise you might unintentionally switch block elements to inline with this class.